gtwrap/matlab_wrapper.py

```diff
@@ -49,6 +49,8 @@ class MatlabWrapper(object):
     }
     """Methods that should not be wrapped directly"""
     whitelist = ['serializable', 'serialize']
+    """Methods that should be ignored"""
+    ignore_methods = ['pickle']
```

Comment:
there is already whitelist?

Reply:
This is because whitelist, although named generically, has been used to determine serialization. 

Ref: https://github.com/borglab/wrap/blob/d37b8a972f6f3aa99a657831102fc22a73b59f21/gtwrap/matlab_wrapper.py#L1529